README.md

```diff
@@ -1,30 +1,33 @@
 # apidoc-plugin-ts
 
-[![Build Status](https://travis-ci.org/tgreyuk/apidoc-plugin-ts.svg?branch=master)](https://travis-ci.org/tgreyuk/apidoc-plugin-ts)
-[![npm](https://img.shields.io/npm/v/apidoc-plugin-ts.svg)](https://www.npmjs.com/package/apidoc-plugin-ts)
-[![Greenkeeper badge](https://badges.greenkeeper.io/tgreyuk/apidoc-plugin-ts.svg)](https://greenkeeper.io/)
+[![Build Status](https://travis-ci.org/geoblink/apidoc-plugin-ts.svg?branch=master)](https://travis-ci.org/geoblink/apidoc-plugin-ts)
+[![npm](https://img.shields.io/npm/v/apidoc-plugin-ts.svg)](https://www.npmjs.com/package/@geoblink/apidoc-plugin-ts)
 
-A plugin for [apidoc](https://www.npmjs.com/package/apidoc) that injects @apiSuccess params from TypeScript interfaces.
+A plugin for [apidoc](https://www.npmjs.com/package/apidoc) that injects `@apiSuccess` params from TypeScript interfaces.
 Supports extended and nested interfaces.
 
 ## Getting started
 
 ```javascript
-npm install --save-dev apidoc apidoc-plugin-ts
+npm install --save-dev apidoc @geoblink/apidoc-plugin-ts
 ```
 
-A custom api-doc param "@apiInterface" is exposed:
+```javascript
+yarn add -D apidoc @geoblink/apidoc-plugin-ts
+```
+
+A custom api-doc param `@apiInterface` is exposed:
 
 ```javascript
-@apiInterface (optional path) {INTERFACE_NAME}
+@apiInterface (optional path to definitions file) {INTERFACE_NAME}
  ```
 
 ## Example
 
 Given the following interface:
 
 ```javascript
-filename: ./employers.ts
+// filename: ./employers.ts
```

Comment:
Because it's not actual code, Github renders it as the first line of the snippet but that's not actually what this is: this is the name of the file so I think it's better to leave it commented so it's rendered as a comment and the snippet can be copy-pasted